R/fromQuantiles.R

```diff
@@ -1,23 +1,8 @@
-# nolint start
-
-#####################################################################################
-## Author: Daniel Sabanes Bove [sabanesd *a*t* roche *.* com]
-## Project: Object-oriented implementation of CRM designs
-##
-## Time-stamp: <[fromQuantiles.R] by DSB Fre 19/06/2015 14:22>
-##
-## Description:
-## Find the best LogisticNormal model for a given set of quantiles at certain
-## dose levels
-##
-## History:
-## 11/02/2014   file creation
-#####################################################################################
-
 ##' @include helpers.R
 ##' @include Model-class.R
 {}
```

Comment:
@danielinteractive  this seems to trigger an error. 
I am not sure why this is used here like this and if putting {} in diffetrnt lines would make sense?